packages/cosmic-swingset/lib/ag-solo/vats/lib-wallet.js

```diff
@@ -145,10 +145,11 @@ export async function makeWallet(
         }),
       ).then(payoutArray =>
         Promise.all(
-          payoutArray.map((payout, payoutIndex) => {
+          payoutArray.map(async (payoutP, payoutIndex) => {
```

Comment:
Is there a way we can write this without an async function in a map?

Reply:
As discussed elsewhere, this is okay for Promise.all arguments.